### ISSUE TITLE
Google Drive sync is broken on 1.11.2

### ISSUE THREAD

**Issue by davidpoza** [keeweb/keeweb#1276]
**Describe the bug**
When we select to syncronize with GDrive, the app prompts for premission, but after fill in info it keeps waiting (spinner animation on database name tab doesn't stop). And never ends the sync process.

**To Reproduce**
1. Clean install frfom deb package. No config files, plugins nor access permissions to google drive are given.
2. Open local database
3. Go database config
4. Click Save to...
5. Select Google Drive
6. Fill in google account info
7. Keeweb keeps trying to sync indefinitely

**Expected behavior**
As usual, for example in previous version 1.11.1. Data gets syncronized with Google Drive at once.

**Environment**
KeeWeb v1.10.1 (07177222, 2019-09-20)
Environment: electron v6.0.7
User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) KeeWeb/1.10.1 Chrome/76.0.3809.139 Electron/6.0.7 Safari/537.36

**Kdbx File**
A personal database, kdbx v4 with argon2: 3 iterations, 1024KB, 1 parallelism

**Logs**
```
[INFO ] 2019-09-29T09:41:06.789Z [app] Started in 340ms ¯\_(ツ)_/¯
[INFO ] 2019-09-29T09:41:06.820Z [updater] Next update check will happen in 1s
[INFO ] 2019-09-29T09:41:07.320Z [updater] Checking for update...
[INFO ] 2019-09-29T09:41:07.321Z [transport] GET https://app.keeweb.info/manifest.appcache
[INFO ] 2019-09-29T09:41:07.338Z [transport] Request to https://app.keeweb.info/manifest.appcache without proxy
[INFO ] 2019-09-29T09:41:07.484Z [transport] Response from https://app.keeweb.info/manifest.appcache: 200
[INFO ] 2019-09-29T09:41:07.486Z [updater] Update check: # 2019-09-29:v1.11.2
[INFO ] 2019-09-29T09:41:07.488Z [updater] Next update check will happen in 86400s
[INFO ] 2019-09-29T09:41:07.488Z [updater] You are using the latest version
[INFO ] 2019-09-29T09:41:18.888Z [open:xxxxx] File open request
[INFO ] 2019-09-29T09:41:18.888Z [open:xxxxx] Open file from supplied content
[DEBUG] 2019-09-29T09:41:18.948Z [argon2] WebAssembly runtime loaded 54ms
[DEBUG] 2019-09-29T09:41:18.965Z [argon2] Hash computed 16ms
[INFO ] 2019-09-29T09:41:19.347Z [file] Opened file xxxxx: 457ms, P=1&I=3&M=1048576&V=19, 100 kB
[DEBUG] 2019-09-29T09:41:19.347Z [app] Add last open file 95c005e8-82a7-6404-a0bb-4bea352a7e7d xxxxx file /xxxxx.kdbx undefined
[DEBUG] 2019-09-29T09:41:19.606Z [storage-file] Watch dir /xxxxx
[DEBUG] 2019-09-29T09:41:24.855Z [storage-gdrive] OAuth: popup opened
[DEBUG] 2019-09-29T09:41:49.362Z [storage-gdrive] OAuth token received
[DEBUG] 2019-09-29T09:41:49.362Z [storage-gdrive] List
```
At this point dev console shows this error:

``` 
Uncaught TypeError: Cannot convert undefined or null to object
    at Function.entries (<anonymous>)
    at l._xhr (index.html:1)
    at index.html:1
    at l (index.html:1)
    at Object.processReturnToApp (index.html:1)
    at index.html:1
    at CallbacksRegistry.apply (/opt/keeweb-desktop/resources/electron.asar/renderer/callbacks-registry.js:41)
    at /opt/keeweb-desktop/resources/electron.asar/renderer/api/remote.js:281
    at EventEmitter.<anonymous> (/opt/keeweb-desktop/resources/electron.asar/renderer/api/remote.js:271)
    at EventEmitter.emit (events.js:200)
```

**Comment by antelle** [comment]
Hi! Already fixed in ae975cdc2fa83d591fe23fd4b487f975ee1132e4, the hotfix is coming tonight.

**Comment by antelle** [comment]
The fix is rolled out
